jsconfig.json

```diff
@@ -2,6 +2,7 @@
 	"compilerOptions": {
 		"alwaysStrict": true,
 		"checkJs": true,
+		"lib": ["esnext", "dom"],
```

Comment:
This line was added I was getting compiling errors in my VS Code saying `object.hasOwn() does not belong in Object Constructor` but from stack overflowing I found this line as a fix. But let me know if this is a correct fix :)

Reply:
Ah I was wondering why this failed! You're ahead of the game 🙂 `Object.hasOwn()` is only supported in Node.js [16.10.0 and above](https://node.green/#ES2022-features-Object-hasOwn) so think we won't be able to use it reliably until after we drop Node.js 16 support in August 😔 there are definitely some alternatives we could try which will support Node.js 14 and earlier versions of 16

Reply:
Ohhh I see 🥲  Hmm, ok can think of some others then! Thank you!